src/LicenseTextReader.ts

```diff
@@ -85,8 +85,10 @@ class LicenseTextReader {
   ): string | null {
     for (const path of paths) {
       const filePath = this.fileSystem.join(modulePath, path);
-      if (/^licen[cs]e/i.test(path) && !this.fileSystem.isDirectory(filePath)) {
-        return path;
+      if (/^licen[cs]e/i.test(path) || /\blicen[cs]e\b/i.test(path)) {
```

Comment:
I think this is a useful addition to the code. What to you think about including the 'Unlicense' while you're at it?
The topic came up here #131.